support-figma/extended-layout-plugin/src/image-res-module.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { json } from "stream/consumers";
+import * as Utils from "./utils";
+
+const IMAGE_HASH_TO_RES_KEY = "image_hash_to_res";
+
+// Hash to image bytes map
+const imageBytesMap = new Map<string, Uint8Array>();
```

Comment:
Can you rename this to `imageHashToBytesMap` to be consistent?

Reply:
Done.